tests/test_scripts.py

```diff
@@ -79,22 +79,26 @@ def test_main_console(script_mod):
         script_mod.main_console()
 
 
+_rl_agent_loading_configs = {
+    "agent_path": CARTPOLE_TEST_POLICY_PATH,
+    # FIXME(yawen): the policy we load was trained on 8 parallel environments
```

Comment:
@ernestum was running into a similar error restoring PPO policies for preference comparison, we had a discussion [here](https://github.com/HumanCompatibleAI/imitation/pull/445#issuecomment-1163496244)